### PR TITLE
AWSEC2 Metrics - removed links

### PR DESCRIPTION
**Pull request by JainM6**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [x] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/observability/aws/integrations/aws-ec2-metrics.md [hunk 1]
```diff
@@ -9,13 +9,13 @@ description: The Sumo Logic App for AWS EC2 allows you to display and gain insig
 
 ## Metrics Types 
 
-Host metrics are gathered by the open-source [SIGAR library](https://github.com/hyperic/sigar). The metrics that are collected are described in [Host Metrics for Installed Collectors](https://helpstaging.sumologic.com/03Send-Data/Sources/01Sources-for-Installed-Collectors/Host-Metrics-Source#Collected_Metrics).
+Host metrics are gathered by the open-source [SIGAR library](https://github.com/hyperic/sigar). The metrics that are collected are described in Host Metrics for Installed Collectors.
```
Comment (kimsauce) [inline docs/observability/aws/integrations/aws-ec2-metrics.md:12]: Leave link to be changed later

File: docs/observability/aws/integrations/aws-ec2-metrics.md [hunk 2]
```diff
@@ -27,7 +27,7 @@ The JSON parameter is in milliseconds. We recommend 60 seconds (60000 ms) or lon
 
 ### AWS Metadata
 
-Collectors running on AWS EC2 instances can optionally collect AWS Metadata such as EC2 tags to make it easier to search for Host Metrics.  Only one AWS Metadata Source for Metrics is required to collect EC2 tags from multiple hosts. For more information, see [AWS Metadata Source for Metrics](https://helpstaging.sumologic.com/03Send-Data/Sources/02Sources-for-Hosted-Collectors/Amazon-Web-Services/AWS-Metadata-(Tag)-Source).
+Collectors running on AWS EC2 instances can optionally collect AWS Metadata such as EC2 tags to make it easier to search for Host Metrics.  Only one AWS Metadata Source for Metrics is required to collect EC2 tags from multiple hosts. For more information, see AWS Metadata Source for Metrics.
```
Comment (kimsauce) [inline docs/observability/aws/integrations/aws-ec2-metrics.md:30]: extra space

```suggestion
Collectors running on AWS EC2 instances can optionally collect AWS Metadata such as EC2 tags to make it easier to search for Host Metrics. Only one AWS Metadata Source for Metrics is required to collect EC2 tags from multiple hosts. For more information, see AWS Metadata Source for Metrics.
```

Comment (kimsauce) [inline docs/observability/aws/integrations/aws-ec2-metrics.md:30]: leave helpstaging link

**Review by kimsauce — CHANGES_REQUESTED** [review]
Some changes

File: docs/observability/aws/integrations/aws-ec2-metrics.md (hunk 2 above)
Comment (kimsauce) [inline docs/observability/aws/integrations/aws-ec2-metrics.md:30]: close extra space and fix (not delete) legacy link

```suggestion
Collectors running on AWS EC2 instances can optionally collect AWS Metadata such as EC2 tags to make it easier to search for Host Metrics. Only one AWS Metadata Source for Metrics is required to collect EC2 tags from multiple hosts. For more information, see [AWS Metadata Source for Metrics](/docs/send-data/hosted-collectors/amazon-aws/aws-metadata-tag-source].
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
changes

File: docs/observability/aws/integrations/aws-ec2-metrics.md (hunk 2 above)
Comment (JainM6) [reply docs/observability/aws/integrations/aws-ec2-metrics.md]: The link can't be fixed now, we don't have the updated page. I will update it once I get the final update from PM and engineer.

**Review by kimsauce — APPROVED** [review]

**Pull request merged**